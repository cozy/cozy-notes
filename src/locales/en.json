{
  "Notes": {
    "Add": {
      "add_note": "Add a note"
    },
    "Delete": {
      "delete_note": "Delete",
      "deleted": "The note has been moved to the trash of your Drive",
      "failed": "The note could not be deleted, please try again"
    },
    "EditorView": {
      "main_placeholder": "Write anythingâ€¦"
    },
    "Editor": {
      "title_placeholder": "Here is your title",
      "exit_confirmation_message": "Automatic saving has not taken into account the latest changes in your doc. Please stay on your current note if you want to keep them.",
      "exit_confirmation_title": "Abandon your changes?",
      "exit_confirmation_leave": "Leave without saving",
      "exit_confirmation_cancel": "Cancel",
      "read_only": "This note is in read-only mode."
    },
    "Files": {
      "share": {
        "cta": "Share"
      }
    },
    "List": {
      "latest_notes": "Latest Notes",
      "name": "Name",
      "updated_at": "Updated at",
      "location": "Location",
      "sharings": "Sharings",
      "at": "%{date} at %{time}"
    },
    "Empty": {
      "welcome": "Welcome on Cozy Notes",
      "after_created": "Once your Note is created, you can find it in your "
    },
    "SavingIndicator": {
      "saved": {
        "default": "Up to date",
        "just_now": "Up to date",
        "secs_ago": "Up to date (saved %{time} sec. ago)",
        "min_ago": "Up to date (saved %{time} min. ago)",
        "mins_ago": "Up to date (saved %{time} min. ago)",
        "hour_ago": "Up to date (saved %{time} hour ago)",
        "hours_ago": "Up to date (saved %{time} hours ago)",
        "day_ago": "Up to date (saved %{time} day ago)",
        "days_ago": "Up to date (saved %{time} days ago)",
        "month_ago": "Up to date (saved %{time} month ago)",
        "monthes_ago": "Up to date (saved %{time} monthes ago)",
        "year_ago": "Up to date (saved %{time} year ago)",
        "years_ago": "Up to date (saved %{time} years ago)"
      },
      "saving": "Saving...",
      "still_saving": "Saving......",
      "out_of_sync": {
        "default": "Saved (off-line)",
        "just_now": "Saved (off-line)",
        "secs_ago": "Saved (off-line for %{time} sec)",
        "min_ago": "Saved (off-line for %{time} min)",
        "mins_ago": "Saved (off-line for %{time} min)",
        "hour_ago": "Saved (off-line for %{time} hour)",
        "hours_ago": "Saved (off-line for %{time} hours)",
        "day_ago": "Saved (off-line for %{time} day)",
        "days_ago": "Saved (off-line for %{time} days)",
        "month_ago": "Saved (off-line for %{time} month)",
        "monthes_ago": "Saved (off-line for %{time} monthes)",
        "year_ago": "Saved (off-line for %{time} year)",
        "years_ago": "Saved (off-line for %{time} years)"
      },
      "saving_impossible": {
        "default": "Changes unsaved (off-line)",
        "just_now": "Changes unsaved (off-line)",
        "secs_ago": "Changes unsaved (off-line for %{time} sec)",
        "min_ago": "Changes unsaved (off-line for %{time} min)",
        "mins_ago": "Changes unsaved (off-line for %{time} min)",
        "hour_ago": "Changes unsaved (off-line for %{time} hour)",
        "hours_ago": "Changes unsaved (off-line for %{time} hours)",
        "day_ago": "Changes unsaved (off-line for %{time} day)",
        "days_ago": "Changes unsaved (off-line for %{time} days)",
        "month_ago": "Changes unsaved (off-line for %{time} month)",
        "monthes_ago": "Changes unsaved (off-line for %{time} monthes)",
        "year_ago": "Changes unsaved (off-line for %{time} year)",
        "years_ago": "Changes unsaved (off-line for %{time} years)"
      }
    },
    "OfflineIndicator": {
      "message": "You're not connected to internet anymore. Your changes will be saved once you will be connected again."
    }
  },
  "Error": {
    "unshared_title": "This share is no longer active",
    "unshared_text": "The owner may have revoked this sharing. You can no longer edit a note at this address.",
    "loading_error_title": "The note couldn't be loaded",
    "loading_error_text_noReturnUrl": "The note may not exists anymore, or our servers may have a temporary glitch. Please try again later.",
    "loading_error_text_returnUrl": "The note may not exists anymore, or our servers may have a temporary glitch. Please go <a href='%{url}'>back</a> and try again later.",
    "422": "Only images are accepted",
    "unknown_error": "An unexpected error occured",
    "only_one_image": "You cannot drag and drop multiple pictures at the same time"
  },
  "manifest": {
    "short_description": "Cozy Notes is your personal and collaborative note-taking application.",
    "long_description": "Notes is an online text editor that can already offer you to:\n\n* Create your own notes with the essential features (bold, underlined, links, tables...)\n* Fill and find your notes with you other documents in files of your personal Drive\n* Edit easily your notes from all your devices\n* Access your notes offline\n* For the fans, use shortcuts and input in markdown format ... and even write with contacts together on the same note (coming soon)!",
    "screenshots": [
      "screenshots/en/screenshot01.png",
      "screenshots/en/screenshot02.png"
    ]
  }
}
