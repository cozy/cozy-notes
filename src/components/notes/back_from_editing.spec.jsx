import React from 'react'
import { I18n } from 'cozy-ui/transpiled/react/I18n'
import { mount } from 'enzyme'
import en from '../../locales/en.json'
import BackFromEditing from './back_from_editing'

describe('BackFromEditing', () => {
  it('mounts', () => {
    const returnUrl = '/'
    const requestToLeave = jest.fn()
    const file = {
      _id: 'uid',
      _type: 'io.cozy.files',
      attributes: {
        dir_id: 'parentuid'
      }
    }
    const component = mount(
      <BackFromEditing
        requestToLeave={requestToLeave}
        returnUrl={returnUrl}
        file={file}
      />,
      {
        wrappingComponent: I18n,
        wrappingComponentProps: {
          lang: 'en',
          dictRequire: () => en
        }
      }
    )
    expect(component.html()).toMatchSnapshot()
  })
})
