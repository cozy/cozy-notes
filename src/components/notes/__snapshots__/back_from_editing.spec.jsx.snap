// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackFromEditing mounts 1`] = `
<div>
  <a
    aria-disabled="false"
    class="MuiButtonBase-root MuiIconButton-root large"
    href="/"
    tabindex="0"
  >
    <span
      class="MuiIconButton-label"
    >
      <svg
        class="styles__icon___23x3R"
        height="16"
        viewBox="0 0 16 16"
        width="16"
      >
        <path
          d="M16 7H3.83l5.59-5.59L8 0 0 8l8 8 1.41-1.41L3.83 9H16V7z"
        />
      </svg>
    </span>
    <span
      class="MuiTouchRipple-root"
    />
  </a>
</div>
`;
